### PR TITLE
TF-1813 Manage the sending queue

### PR DESCRIPTION
**Pull request by dab246**
### Issue

#1813 

### Resolved

https://github.com/linagora/tmail-flutter/assets/80730648/5b12a666-0b0c-4369-a7b1-3549feb2757e

https://github.com/linagora/tmail-flutter/assets/80730648/70207263-7d52-4f3e-a7d1-3a6c47d12f94

### CODE REVIEW

**Comment by chibenwa** [comment]
We are not yet sending the emails when coming back online, right?

**Comment by dab246** [comment]
> We are not yet sending the emails when coming back online, right?

Don't worry this ticket #1842  is being worked on.

File: lib/features/base/mixin/compose_floating_button_mixin.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import 'package:core/presentation/resources/image_paths.dart';
+import 'package:core/presentation/utils/style_utils.dart';
+import 'package:core/presentation/views/floating_button/scrolling_floating_button_animated.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+import 'package:tmail_ui_user/main/routes/route_navigation.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+mixin ComposeFloatingButtonMixin {
```
Comment (sherlockvn) [inline lib/features/base/mixin/compose_floating_button_mixin.dart:12]: Why do you have to use mixin here, why just use StatelessWidget ?

Comment (dab246) [reply lib/features/base/mixin/compose_floating_button_mixin.dart:12]: Oh, I forgot to fix it. We can use it as a stateless

**Pull request merged**